### PR TITLE
Add some Heroku and Travis hints, fix typos

### PR DESCRIPTION
**Pull request by snappy316**

### CODE REVIEW

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ https://devcenter.heroku.com/articles/getting-started-with-rails4
 
 * One really important step that is left out of that article is that you need to tell Rails that it is ok to serve static assets, which Rails 4 will not do by default. If you don't do that, your website images will seem to vanish over time.
 
-* Find this setting in the Production Environment config file, which is used on Heroku: `config/environments/production.rb`. Change that value to `true`.
+* Find these settings in the Production Environment config file (`config/environments/production.rb`), which is used on Heroku, and set them to true - `config.serve_static_files` and `config.assets.digest`
```
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: Hmmm, `config.assets.digest` is set to true by default: http://guides.rubyonrails.org/asset_pipeline.html#in-production .

We might not want to include that part of the instruction.

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md [hunk 2]
```diff
@@ -43,6 +42,8 @@ https://devcenter.heroku.com/articles/getting-started-with-rails4
 
       $ heroku run rake db:migrate
 
+* If you're using Google Fonts and run into issues with it loading, either change the protocol from `http://fonts...` to `https://fonts...`, or leave the protocol out and just do `//fonts...`
+
```
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: This one is pretty specific to Google, or the web in general, not Heroku.  The book doesn't suggest to use Google fonts, so I don't think it's time to include this yet.

**Comment by brookr** [comment]
Thanks, Dave!

**Comment by sealocal** [comment]
:+1:

File: chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md (hunk 2 above)
Comment (sealocal) [inline chapter_4_rails_theming,_continuous_integration,_and_deployment/deploy_to_heroku.md]: Will add back based on brookr's note.

**Pull request merged**